src/lucky/action_callbacks.cr

```diff
@@ -24,14 +27,64 @@ module Lucky::ActionCallbacks
     \{% end %}
   end
 
+  # Run a method before an action starts
+  #
+  # Methods will run in the order that each `before` is defined. Also, each
+  # method must return a `Lucky::Response` such as `redirect` or `render`, or
```

Comment:
Rather than having 2 `or`s I think it'd look cleaner to have `redirect`, `render`, or ...